soh/soh/Enhancements/debugger/debugSaveEditor.cpp

```diff
@@ -611,6 +616,17 @@ void DrawInventoryTab() {
                     if (ImGui::ImageButton(SohImGui::GetTextureByName(slotEntry.name), ImVec2(32.0f, 32.0f),
                                            ImVec2(0, 0), ImVec2(1, 1), 0)) {
                         gSaveContext.inventory.items[selectedIndex] = slotEntry.id;
+                        // Set adult trade item flag if you're playing adult trade shuffle in rando
+                        if (gSaveContext.n64ddFlag &&
+                            OTRGlobals::Instance->gRandomizer->GetRandoSettingValue(RSK_SHUFFLE_ADULT_TRADE) &&
```

Comment:
you can use `Randomizer_GetSettingValue` instead of going through `gRandomizer` here

Reply:
nevermind, seems that's not the case, this is good as is